### DISCUSSION TITLE
Why failed to start alist?

### DISCUSSION THREAD

**Discussion by l21565** [AlistGo/alist#2295] [Q&A]
HardWare: Synology DS212j，ARMv5TE
OS: DSM 6.2.4
alist release: alist-linux-arm-5 v3.3.0
 ./alist server
INFO[2022-11-11 11:24:56] reading config file: data/config.json
INFO[2022-11-11 11:24:56] config file not exists, creating default config file
INFO[2022-11-11 11:24:56] load config from env with prefix: ALIST_
INFO[2022-11-11 11:24:56] init logrus...
INFO[2022-11-11 11:24:56] Successfully created the admin user and the initial password is: xxxxxx
INFO[2022-11-11 11:24:57] Aria2 not ready.
INFO[2022-11-11 11:24:57] start server @ 0.0.0.0:5244
FATA[2022-11-11 11:24:57] failed to start: accept tcp [::]:5244: accept: function not implemented

**Comment by xhofe** [comment]
Consider using the musl version as an alternative due to the version of the glibc on your platform is too low

**Comment by l21565** [comment]
Thx for the replay.I've tried the musl version,but it can't run on the DSM of DS212j.



---- Replied Message ----
| From | Noah ***@***.***> |
| Date | 11/12/2022 22:59 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [alist-org/alist] Why failed to start alist? (Discussion #2295) |

Consider using the musl version as an alternative due to the version of the glibc on your platform is too low
